amplify/backend/auth/vocelappcf1453cb/cli-inputs.json

```diff
@@ -0,0 +1,68 @@
+{
+  "version": "1",
+  "cognitoConfig": {
+    "identityPoolName": "vocelappcf1453cb_identitypool_cf1453cb",
```

Comment:
Do we have instructions on how to change these for a different AWS backend instance?